pyemma/msm/estimators/maximum_likelihood_msm.py

```diff
@@ -474,76 +305,6 @@ def active_count_fraction(self):
         return float(_np.sum(hist_active)) / float(_np.sum(hist))
 
     ################################################################################
-    # For general statistics
-    ################################################################################
-
-    def trajectory_weights(self):
```

Comment:
Isn't that general? We only distribute the microstate stationary distribution to frames in those microstates. Can't we use this in the OOMReweighted case?